components/libraryprovider.py

```diff
@@ -24,12 +23,81 @@
 #     'name': 'dav1d'
 # },
 # 'updatebot': {
-#     'enabled': True,
 #     'maintainer-bz': 'nobody@mozilla.com',
-#     'maintainer-phab': 'nobody'
```

Comment:
Amazed at myself for getting the variable name styling for `maintainer-bz` and the like right here, but literally nowhere else. Thanks for cleaning this up!